### PR TITLE
feat(cards): Horizontal blocks.

### PR DESCRIPTION
**Pull request by sgomes**
Adds support for horizontal blocks in cards, used for composition.

@Garbee and/or @traviskaufman PTAL!

### CODE REVIEW

File: packages/mdl-card/mdl-card.scss [hunk 1]
```diff
@@ -105,6 +106,22 @@
...
   }
 
+  &__media-item {
+    display: inline-block;
+    width: auto;
+    height: 80px;
+    margin: 16px 0 0;
+    padding: 0;
+
+    @each $mult, $name in (1: "1", 1.5: "1dot5", 2: "2", 3: "3") {
```
Comment (Garbee) [inline packages/mdl-card/mdl-card.scss]: It seems to me like `1` here is the default, so what scenario necessitates including a modifier for it?

Comment (sgomes) [inline packages/mdl-card/mdl-card.scss]: Nothing in particular, just completeness. Happy to remove it if you think that's best.

Comment (Garbee) [inline packages/mdl-card/mdl-card.scss]: Yea, I'd rather it get removed until a scenario presents itself where this is needed. That way we never ship what isn't clearly necessary.

Comment (sgomes) [inline packages/mdl-card/mdl-card.scss]: Done!

File: packages/mdl-card/mdl-card.scss [hunk 2]
```diff
@@ -105,6 +106,22 @@
...
+  &__media-item {
+    display: inline-block;
+    width: auto;
+    height: 80px;
+    margin: 16px 0 0;
+    padding: 0;
+
+    @each $mult, $name in (1.5: "1dot5", 2: "2", 3: "3") {
+      // postcss-bem-linter: ignore
+      &--#{$name}x {
```
Comment (traviskaufman) [inline packages/mdl-card/mdl-card.scss]: very clever!

**Comment by traviskaufman** [comment]
👍 LGTM!

**Pull request merged**